src/controllers/teamController.ts

```diff
@@ -57,8 +57,28 @@ const checkTeamHappiness = async (
     next(error);
   }
 };
+
+const getTeamInfo = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const { teamId } = req.params;
+
+  try {
+    const data = await teamService.getTeamInfo(+teamId);
+
+    return res
+      .status(statusCode.OK)
+      .send(success(statusCode.OK, message.GET_TEAM_INFO, data));
```

Comment:
```suggestion
      .send(success(statusCode.OK, message.READ_TEAM_INFO, data));
```
READ_TEAM_INFO가 좋을거 같아요~